ajax/libs/howler/package.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "howler",
   "filename": "howler.min.js",
-  "version": "1.1.29",
+  "version": "2.0.0",
```

Comment:
Version can not be changed since there is no v2 files in the repo, you can leave it alone or add the v2.0.0 files to fix it.